pkg/provider/pulumi/aws/aws.go

```diff
@@ -98,12 +98,12 @@ func New(p *project.Project, name string, envMap map[string]string) (common.Pulu
 
 	// Hydrate from file if already exists
 	b, err := os.ReadFile(filepath.Join(p.Dir, "nitric-"+name+".yaml"))
-	if !errors.Is(err, os.ErrNotExist) {
+	if err != nil && !errors.Is(err, os.ErrNotExist) {
```

Comment:
good catch, but probably less messy to test errors.Is()

Reply:
fixed